wsi/drm/drmdisplaymanager.h

```diff
@@ -86,6 +86,7 @@ class DrmDisplayManager : public HWCThread, public DisplayManager {
   std::shared_ptr<DisplayHotPlugEventCallback> callback_ = NULL;
   std::unique_ptr<NativeBufferHandler> buffer_handler_;
   GpuDevice *device_ = NULL;
+  volatile bool ignore_updates_ = false;
```

Comment:
@shuangwan01 The volatile modifier is unused in this project except in: tests/third_party/... and in libtool and completely missing from any source files. I understand why you would want to use it here, but I would rather you implement your own synchronization instead.

Reply:
@stripes416  Thanks for comment. Will remove the usage of volatile since it's just for avoiding potential thread synchronization issue. So far no thread sync issue found during testing.